dwl.c

```diff
@@ -770,6 +773,10 @@ commitlayersurfacenotify(struct wl_listener *listener, void *data)
 			&layersurface->link);
 		layersurface->layer = wlr_layer_surface->current.layer;
 	}
+
+	// Damage the whole screen
+	if (selmon)
+		wlr_output_damage_add_whole(selmon->damage);
```

Comment:
I think you mean `m` instead of `selmon`, right?  I originally intended that neither `selmon` nor `c->mon` nor `output->data` would ever be null - it's possible that the output-management changes affected that invariant, but if not, there's no need to do this conditionally.

Reply:
Ah, completely missed that. Resolved in new revision.